packages/core/stories/layout/grid-layout.doc.stories.mdx

```diff
@@ -42,7 +39,7 @@ You want your layout to flow in a single dimension. In this case we recommend yo
 
 #### Grid Item inside default grid layout
 
-A grid item allows more complex controls to the default behavior of a grid child, like defining how many rows or columns will it span trough and it's alignment
+A grid item allows more complex controls to the default behavior of a grid child, like defining how many rows or columns will it span trough, and it's alignment.
```

Comment:
```suggestion
A grid item allows more complex controls to the default behavior of a grid child, like defining how many rows or columns it will span, and its alignment.
```